modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing an upstream vSphere CSI Operator Driver
+
+If you have installed an upstream vSphere Container Storage Interface (CSI) Operator Driver, and then try to update your cluster to {product-title} 4.10 or later, {product-title} prevents the update.
+
+To correct this issue and update the cluster, a cluster admin must:
+
+. Delete the upstream vSphere CSI driver Deployment object.
+. Delete the upstream vSphereCSI driver Daemonset objects:
+.. Be sure to delete `CSIDriver`:
```

Comment:
@lpettyjo The step `delete CSIDriver object` and `delete configmap and secret objects` is not  step2's sub steps. From my side, the steps ownership looks a bit strange. It seems better change to:
```
1. Delete the upstream vSphere CSI driver(VMware vSphere Container Storage Plug-in) Deployment and  Daemonset objects.

2. Delete configmap and secret objects which created previously used by the upstream vSphere CSI Driver.

3.  Delete the upstream vSphereCSI driver `CSIDriver` object.
~ $ oc delete CSIDriver csi.vsphere.vmware.com
csidriver.storage.k8s.io "csi.vsphere.vmware.com" deleted

```
WDYT?